conda.recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ requirements:
     - pip
     - setuptools
     - click
-    - h5py
+    - h5py>=3.1.0
```

Comment:
I tried this in rex and it failed to build, turns out Conda is still at 2.1

Reply:
ah i see. Fixed